src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/StatusColumn.java

```diff
@@ -23,7 +22,7 @@ public FlowNodeViewColumn getDefaultInstance() {
 
         @Override
         public String getDisplayName() {
-            return Messages.StatusColumn_DisplayName();
+            return "Status";
```

Comment:
Why are we de-internationalizing here?

Reply:
(BTW this was already noted in https://github.com/jenkinsci/workflow-support-plugin/pull/60 where this change was introduced.)

Reply:
@jglick You noted it was due to the parent POM update (which I saw0, but the fully explanation gives that context needed. 

Ideally we'd add it to a Messages file for this plugin if that is restricted (just that particular messages file, I hope?) but I'll let this one go - not a big deal.

Reply:
Generally, there are lots of strings in Pipeline-related plugins that _could_ be made localizable but currently are not.